### ISSUE TITLE
Version 1.2.0 broke the app

### ISSUE THREAD

**Issue by dsh-sportsbiz** [aws/aws-sdk-js-crypto-helpers#214]
Cannot even start the app now

convertToBuffer.js:17 Uncaught ReferenceError: Buffer is not defined
    at Object.52023 (convertToBuffer.js:17)
    at __webpack_require__ (bootstrap:19)
    at Object.50007 (index.js:9)
    at __webpack_require__ (bootstrap:19)
    at Object.52897 (webCryptoSha256.js:11)
    at __webpack_require__ (bootstrap:19)
    at Object.52809 (crossPlatformSha256.js:11)
    at __webpack_require__ (bootstrap:19)
    at Object.48464 (index.js:8)
    at __webpack_require__ (bootstrap:19)

**Comment by KeithGillette** [comment]
Same issue with our project attempting to update from `1.1.1`:
```
Uncaught ReferenceError: Buffer is not defined
    at Object.67107 (convertToBuffer.js:8)
    at __webpack_require__ (bootstrap:24)
    at fn (hot module replacement:61)
    at Object.88734 (index.js:6)
    at __webpack_require__ (bootstrap:24)
    at fn (hot module replacement:61)
    at Object.53413 (webCryptoSha256.js:4)
    at __webpack_require__ (bootstrap:24)
    at fn (hot module replacement:61)
    at Object.89818 (crossPlatformSha256.js:5)
```

**Comment by florinbardosi** [comment]
Small fix on your side, crash on ours. Do you have a quick fix? This shouldn't happen if someone is using fixed versions.

**Comment by florinbardosi** [comment]
Adding the following as dependencies will fix the build until the fix is provided

		"@aws-crypto/sha256-browser": "1.1.1",
		"@aws-crypto/sha256-js": "1.1.0",

**Comment by dsh-sportsbiz** [comment]
> Small fix on your side, crash on ours. Do you have a quick fix? This shouldn't happen if someone is using fixed versions.
> 
> convertToBuffer.js?61c1:8 Uncaught ReferenceError: Buffer is not defined
> at eval (convertToBuffer.js?61c1:8)
> at Object../node_modules/@aws-crypto/util/build/convertToBuffer.js (main.js:1418)
> at **webpack_require** (main.js:790)
> at fn (main.js:101)
> at eval (index.js?a54d:6)
> at Object../node_modules/@aws-crypto/util/build/index.js (main.js:1426)
> at **webpack_require** (main.js:790)
> at fn (main.js:101)
> at eval (jsSha256.js?0103:7)
> at Object../node_modules/@aws-crypto/sha256-browser/node_modules/@aws-crypto/sha256-js/build/jsSha256.js (main.js:1338)

Yes, going back to prior version, as you said. I keep unfixed latest versions in dev as it helps finding these issues quickly:-) And stay update of course

**Comment by seebees** [comment]
this should be fixed by #217 that was released in v1.2.1
